app.js

```diff
@@ -103,34 +103,24 @@ function updateMetronome() {
 let isPlaying = false;
 let timeout;
 
+let lastClick = performance.now();
 function playMetronome() {
-  console.log("click");
+  const now = performance.now();
+  const expected = 60000 / tempo;
+  const actual = now - lastClick;
+  const diff = new Intl.NumberFormat("en-US", { signDisplay: "exceptZero" }).format(actual - expected);
+  console.log("click", { expected, actual, diff });
+  lastClick = now;
   timeout = setTimeout(playMetronome, 60000 / tempo);
 }
 
 startStopButton.addEventListener("click", () => {
+  startStopButton.classList.toggle("stopped");
+  isPlaying = !isPlaying;
```

Comment:
The changes here are just to improve readability. If we can pull these out of the if-else block, let's. (If there were more complex logic where, say, four variables got updated or not, I'd suggest the opposite change!)